src/common/options/rgw.yaml.in

```diff
@@ -1955,6 +1955,66 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_data_sync_poll_interval
+  type: int
+  level: dev
+  default: 20
+  fmt_desc: Once multisite's incremental sync of a datalog shard is caught up
+    with its source, it will wait this long (in seconds) before polling for
+    more changes.
+  services:
+  - rgw
+  see_also:
+  - rgw_meta_sync_poll_interval
+  with_legacy: true
+- name: rgw_meta_sync_poll_interval
+  type: int
+  level: dev
+  default: 20
+  fmt_desc: Once multisite's incremental sync of a mdlog shard is caught up
+    with its source, it will wait this long (in seconds) before polling for
+    more changes.
+  services:
+  - rgw
+  see_also:
+  - rgw_data_sync_poll_interval
+  with_legacy: true
+- name: rgw_bucket_sync_spawn_window
+  type: int
+  level: dev
+  default: 20
+  fmt_desc: The maximum number of items that bucket sync is willing to
+    process at a time (per remote bilog shard).
```

Comment:
maybe "in parallel" instead of "at a time"?
since it's not clear (to me) what "at a time" means.